src/main.rs

```diff
@@ -204,7 +204,7 @@ impl<'a> Service<'a> {
 
                 // discord requires details to be at least two characters
                 while details.chars().count() < 2 {
-                    details.push('_');
+                    details.push('\u{200B}');
```

Comment:
Can you just add a comment to make it clear what the char is please:

```suggestion
                    details.push('\u{200B}'); // zwsp
```